### DISCUSSION TITLE
About Rapid Blazor

### DISCUSSION THREAD

**Discussion by Pramod1008** [jasontaylordev/CleanArchitecture#853] [Q&A]
I using your Rapid Blazor Project ....When i click on login that time i getting this following error  ..I didn't change anything in your project....

Error-
**There was an error trying to log you in: 'Invalid response Content-Type: text/html, from URL: https://localhost:7124/.well-known/openid-configuration'**

Please give some solution for this.....

**Comment by jasontaylordev** [comment]
Sorry, I've got limited capacity to support Rapid Blazor at the moment. 
If you like, raise an issue on that template and I'll try to get to it soon.

**Comment by Pramod1008** [comment]
Thanks Sir,

Issue resolved...
